modules/tinymce/src/core/test/ts/browser/FormattingCommandsTest.ts

```diff
@@ -135,6 +135,12 @@ describe('browser.tinymce.core.FormattingCommandsTest', () => {
     assert.equal(editor.getContent(), '<p style="text-align: center;">test 123</p>');
     assert.isTrue(editor.queryCommandState('JustifyCenter'), 'should have JustifyCenter state true');
 
+    editor.setContent('<pre>test 123</pre>');
```

Comment:
I wonder if we should start splitting these out into separate it calls. This file has one massive combined test that just gets bigger and bigger.

Reply:
Yeah agreed, this should really be broken up into multiple `it` blocks and potentially even separate files. Hannes do you mind logging a tech debt task to do that?